.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
 go_import_path: github.com/open-telemetry/opentelemetry-collector
 
 go:
-  - 1.13.x
+  - 1.14.x
```

Comment:
Is this a necessary part of the PR? Is there a particular new feature in 1.14 that this relies on? We need to announce to all contributors to update go version locally if this is needed.

Reply:
I don't believe a specific 1.14 feature is relied upon, aside from the fact that this go.mod was generated with golang's 1.14 dependency fetching behavior (see https://golang.org/doc/go1.14#incompatible-versions)

Otherwise someone running `go get` etc may naively break the build again.

However, the upgrade to 1.14 slightly "regresses" one of the tests which I've fixed below. But now 1.13 will fail because Go has changed their core library error reporting between 1.13 and 1.14.

Reply:
If go 1.14 is not required I think it is better to revert that part to merge this PR quicker. 
To upgrade to newer go version our policy is to at least make an announcement in gitter and give contributors a bit time to notice it.

Reply:
I think changing to 1.14 is fine, but maybe in a separate PR? I can help with that :)